### ISSUE TITLE
Use in other places

### ISSUE THREAD

**Issue by dabreegster** [ITSLeeds/MinorRoadTraffic#9]
Hi! I've seen the Scotland-wide output data for this. I was wondering

1) About how hard would it be to produce similar results for England?
2) What input data is required -- following the Scotland notebook, very unclear about that

Thanks!

**Comment by mem48** [comment]
@juanfonsecaLS1  I think has the best advice on this

See also https://github.com/itsleeds/MinorRoadTraffic/blob/main/vignettes/IOW.Rmd and https://github.com/itsleeds/MinorRoadTraffic/blob/main/vignettes/Oxford.Rmd which are worked examples for Isle of White and Oxford. 

Note that the quality of the forecast is not great. I run this as a Master's topic with students making incremental progress each year.

**Comment by juanfonsecaLS1** [comment]
The model used for Scotland was adapted from those worked examples. 

1. It wouldn't be too hard but a good dataset with traffic on minor roads is needed for training the model. We used some counts published as part of the 20 mph scheme.
2. It uses OS open roads, and traffic counts (DfT road traffic statistics and 20mph-scheme dataset)

@dabreegster I think you have access to the npt repository. Take a look at the pdf in "nptscot/scottraffic/tree/main/technical_note"

**Comment by dabreegster** [comment]
Thanks both for the clarification! Sounds like we'd first need to find equivalent minor road volumes for training.

**Comment by Robinlovelace** [comment]
Yes, all about a good training dataset. If anyone finds anything in that direction please let us know here.
